galter_subjects_utils/reader.py

```diff
@@ -34,3 +38,24 @@ def mapping_by(iterable, by, keys=None):
         d.get(by): {k: d.get(k) for k in keys} if keys else d
         for d in iterable
     }
+
+
+def get_rdm_subjects(scheme):
+    """Return all rdm subjects of corresponding scheme."""
+
+    is_scheme = (
+        text('json::json->>\'scheme\' = :scheme')
+        .bindparams(
+            bindparam(
+                "scheme",
+                value=scheme,
+            )
+        )
+    )
+
+    stmt = (
+        select(SubjectsMetadata.json)
```

Comment:
This was the big memory saver: extracting only the json over a full ORM object makes a notable difference.